release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -109,6 +109,12 @@ The containerized Data Plane Development Kit (DPDK) is now GA in {product-title}
 [id="ocp-4-9-olm"]
 === Operator lifecycle
 
+[id="ocp-4-9-olm-operand-deletion"]
+==== Removing Operands when uninstalling an Operator using the web console
+
+When you uninstall an Operator by using the web console, you can now remove Operands managed by the Operator automatically, including custom resource definitions (CRDs) and custom resources (CRs). For more information, see
```

Comment:
Actually, no. Not include the CRD. Only the custom resources (CRs) will be removed. The custom resource definitions (CRDs) still exist.

Reply:
@kevinrizza I test this point, only CR will be removed, not the CRD. Is it as expected? If not, I think that is a bug.

Reply:
I don't think so, it looks like nothing was ever implemented to delete the CRD itself -- which is reasonable, given the tenancy concerns around deleting operators installed in multiple scopes